core/msg.c

```diff
@@ -187,41 +188,90 @@ int msg_send_to_self(msg_t *m)
     return res;
 }
 
-int msg_send_int(msg_t *m, kernel_pid_t target_pid)
+static int _msg_send_oneway(msg_t *m, kernel_pid_t target_pid, bool *sched)
```

Comment:
can we get rid of the extra sched parameter? I think it is valid t just re-use `sched_context_switch_request`.

Reply:
And it's gone.